src/test/java/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,40 @@
+package strategy;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashMap;
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import strategy.impl.BalanceTransactionImpl;
+import strategy.impl.PurchaseTransactionImpl;
+import strategy.impl.ReturnTransactionImpl;
+import strategy.impl.SupplyTransactionImpl;
+
+class OperationStrategyTest {
```

Comment:
This class should test OperationStrategyImpl, shouldn't it?
So, where is the initiation of this service?